keps/9988-multikueue-manager-quota-automation/README.md

```
@@ -514,7 +568,7 @@ Our initial idea was to let it affect the manager-side `nominalQuota`; however,
 
 Another option is to re-use the concept of "effective quota", or "accessible quota" in terms of [KEP-8826](https://github.com/kubernetes-sigs/kueue/pull/8864); that is, leave `nominalQuota` unchanged but apply the mulitplier on the level of "accessible quota" which would be the one actually enforced. This would allow to keep `nominalQuota` serving [User Story 4](#story-4) while letting the multiplier affect the actual dispatching behavior.
 
-This may combine well with the idea of [moving aggregated quotas out of ClusterQueueSpec](#move-the-aggregated-quota-out-of-clusterqueuespec).
+This may combine well with the idea of moving aggregated quotas out of ClusterQueueSpec (see [KEP-11823]()).
```

markdownlint-cli2 (0.22.1)

[warning] No empty links

(MD042, no-empty-links)